source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
+**Option 1**: Platform has multiple databases and the user can search the whole platform, multiple selected databases, or a single database.
 
-The audit test consists of making an Item_Investigation twice in succession (double-clicks). If the two clicks occur within a 30-second time-span, only the second Total_Item_Investigations made must be recorded. If the two clicks occur with more than 30 seconds between, then 2 Total_Item_Investigations must be counted.
+The auditor MUST run 100 searches on the platform
 
-The auditor must carry out a total of 30 tests, and each test will consist of 2 item investigations. There are 2 types of tests that must be carried out:
+* 50 searches against only 1 selected database, resulting in 50 Searches_Regular against that database.
+* 25 searches against 2 selected databases, resulting in 25 Searches_Regular against both databases.
+* 25 searches against all databases on the platform, resulting in 25 Searches_Regular against every database.
 
-* “Inside” tests (Two item investigations are made to the same item the second item Investigation is made within 30 seconds of the first).
-* “Outside” tests (Two item investigations are made to the same item, and the second item investigation is made more than 30 seconds after the first).
+**Option 2**: Platform has multiple databases and the user can search the whole platform or a single database.
 
-The auditor must carry out 15 **inside** tests.
+The auditor MUST run 100 searches on the platform
 
-This must result in:
+* 50 searches against only 1 selected database, resulting in 50 Searches_Regular against that database.
+* 50 searches against all databases on the platform, resulting in 50 Searches_Regular against every database.
 
-* 15 Total_Item_Investigations being reported in the DR_D1 Standard View.
+**Option 3**: Platform has multiple databases but the user can only search the whole platform.
 
-The auditor must carry out 15 **outside** tests.
+The auditor MUST run 100 searches, resulting in 100 Searches_Automated (1 per search).
 
-This must result in:
+**Option 4**: Platform has multiple databases but the user can only search a single database at a time.
 
-* 30 Total_Item_Investigations being reported in the DR_D1 Standard View.
+The auditor MUST run 100 searches across at least 10% of the available databases, restricted to one database at a time, resulting in 100 Searches_Regular (1 per search).
 
+**Option 5**: Platform has only one database.
 
-E.2.2.3 Standard View: DR_D2
-''''''''''''''''''''''''''''
+The auditor MUST run 50 searches, resulting in 50 Searches_Regular (1 per search).
 
-Databases Access Denied: This Standard View reports on access-denied activity for databases where a user is denied access because simultaneous user licenses are exceeded or the institution does not have a license for the database.
 
-An audit of this Standard View and related tests requires the following:
+E.2.3.2 Total_Item_Requests
```

Comment:
The titles of this and the next section only mention Total_Item metrics, but they also cover Unique_Item metrics. Is it intentional that the Unique_Title metrics aren't covered?